PLUB/Sources/Views/Home/BookmarkViewController.swift

```diff
@@ -188,3 +223,15 @@ extension BookmarkViewController: SelectedCategoryChartCollectionViewCellDelegat
     viewModel.tappedBookmark.onNext(plubbingID)
   }
 }
+
+extension BookmarkViewController: BookmarkHeaderViewDelegate {
+  func didTappedInterestListChartButton() {
+    selectedCategoryType = .chart
+    interestListCollectionView.reloadData()
+  }
+  
+  func didTappedInterestListGridButton() {
+    selectedCategoryType = .grid
+    interestListCollectionView.reloadData()
+  }
+}
```

Comment:
layout만 변경하시는 거라면 reloadData보다는 `invalidateLayout()`을 권장합니다.

```suggestion
  func didTappedInterestListChartButton() {
    selectedCategoryType = .chart
    interestListCollectionView.collectionViewLayout.invalidateLayout()
  }
  
  func didTappedInterestListGridButton() {
    selectedCategoryType = .grid
    interestListCollectionView.collectionViewLayout.invalidateLayout()
  }
}
```

Reply:
헐 처음 보는 함수인데 바로 수정해보도록 하겠습니다, 바로 공부해봐야겠네요 감사합니다 승현님 👍👍👍